include/Graph/Graph.hpp

```diff
@@ -524,16 +539,15 @@ class Graph {
    * @param workingDir The path to the directory in which is placed the Input
    * file
    * @param OFileName The Input File Name ( )
-   * @param compress Indicates if the Input is compressed
    * @param readNodeFeat Indicates if import also Node Features
    * @param readEdgeWeight Indicates if import also Edge Weights
    * @return 0 if all OK, else return a negative value
    */
   virtual int readFromFile(
       InputOutputFormat format = InputOutputFormat::STANDARD_CSV,
       const std::string &workingDir = ".",
-      const std::string &OFileName = "graph", bool compress = false,
-      bool readNodeFeat = false, bool readEdgeWeight = false);
+      const std::string &OFileName = "graph", bool readNodeFeat = false,
+      bool readEdgeWeight = false);
```

Comment:
⚠️ Public API change. We're removing the ability to compress. Users can use their own compression wrappers if they want.